crates/signer/Cargo.toml

```diff
@@ -30,7 +30,7 @@ eth-keystore = { version = "0.5.0", default-features = false, optional = true }
 
 # mnemonic
 coins-bip32 = { version = "0.8.7", default-features = false, optional = true }
-coins-bip39 = { version = "0.8.7", default-features = false, optional = true }
+coins-bip39 = { version = "0.8.7", optional = true }
```

Comment:
This enables all languages: https://docs.rs/crate/coins-bip39/latest/features
Can we just enable the english one?

Reply:
probably want to introduce an `all-bip39-lists` feature that re-exports all wordlists, while keeping english as default